lms/djangoapps/labster_course_license/views.py

```diff
@@ -58,6 +60,36 @@ def license_handler(request, course, ccx=None):
     return HttpResponseBadRequest('Only GET and POST methods are supported.')
 
 
+class LicensedSimulationsUpdateView(APIView):
+    """
+    Labster License update handlers.
+    """
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
```

Comment:
Probably `IsAdminUser` better idea to user, otherwise everyone can use this view.